governance/votes/Executive vote - July 2, 2021.md

```diff
@@ -0,0 +1,144 @@
+
+This executive proposal **does not** include an office-hours modifier and can be executed immediately after the GSM Pause Delay has expired.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### July Core Unit Budget Distributions
+
+Due to the positive outcomes of multiple [Core Unit Budget Proposals](https://mips.makerdao.com/mips/list?&search=$AND(%23core-unit,%20%23budget,%20%23active)&subproposalsMode=true), the following budget allocations will be made to fund operations in the month of July if this executive proposal passes.
+
+**155,000 DAI** will be distributed to the **[Real World Finance (RWA-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1)**  core unit at **0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP12).
+
+**182,000 DAI** will be distributed to the **[Risk (RISK-001)](https://mips.makerdao.com/mips/details/MIP39c2SP2)**  core unit at **0xd98ef20520048a35EdA9A202137847A62120d2d9**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP13).
+
+**273,334 DAI** will be distributed to the **[GovAlpha (GOV-001)](https://mips.makerdao.com/mips/details/MIP40c3SP11)** core unit at **0x01D26f8c5cC009868A4BF66E268c17B057fF7A73**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP3). Note that this includes the GovAlpha continuity budget.
+
+**510,000 DAI** will be distributed to the **[Protocol Engineering (PE-001)](https://mips.makerdao.com/mips/details/MIP39c2SP7)** core unit at **0xe2c16c308b843eD02B09156388Cb240cEd58C01c** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7).
+
+**126,117 DAI** will be distributed to the **[Growth (GRO-001)](https://mips.makerdao.com/mips/details/MIP39c2SP4)**  core unit at **0x7800C137A645c07132886539217ce192b9F0528e**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP4).
+
+**44,375 DAI** will be distributed to the **[Content Production (MKT-001)](https://mips.makerdao.com/mips/details/MIP39c2SP5)**  core unit at **0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP41c4SP5) and modified within this forum [post](https://forum.makerdao.com/t/adding-content-production-multisig/8428/5).
+
+**419,677 DAI** will be distributed to the **[Oracles (ORA-001)](https://mips.makerdao.com/mips/details/MIP39c2SP13)** core unit operational wallet at **0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP15).
```

Comment:
Was confused where this figure came from at first, the budget MIP is a little unclear IMO. Might have Pablo update it later

Reply:
Nik updated it on the forum already, he said he was going to do a PR for the repo version too.